vagrant/Vagrantfile

```diff
@@ -63,15 +75,37 @@ Vagrant.configure("2") do |config|
     adduser $SIRFUSERNAME sudo
     { echo $SIRFPASS; echo $SIRFPASS; } | passwd $SIRFUSERNAME 
 
-    apt-get update
-    apt-get install -y wget git xorg xterm gdm menu gksu synaptic gnome-session gnome-panel metacity --no-install-recommends
+   apt-get update
+	# mark grup as hold as it requires user intervention on upgrade!
+	apt-mark hold grub-pc
+	# upgrade the list of packages
+	apt-get upgrade -y
+	#install upgrades
+	apt-get install -y wget git xorg xterm gdm menu gksu synaptic gnome-session gnome-panel metacity --no-install-recommends
     apt-get install -y at-spi2-core gnome-terminal gnome-control-center nautilus 
     service gdm start
-
-    # To hide vagrant from login screen:
+    
+	# set the current locale, otherwise the gnome-terminal doesn't start
+	sudo locale-gen en_GB.UTF-8
+	sudo update-locale LANG=en_GB.UTF-8
+	sudo localectl set-x11-keymap us
```

Comment:
are you sure about using a us keyboard? Might be why I had to install a UK one. probably a good idea to add a few locales/keyboard maps from the beginning (like you did with portuguese at the time) and just tell people how to switch.

Reply:
Which locale would you want on board?

I though US keymap would be the most common. 
Which one should I use?